lib/filters.js

```diff
@@ -0,0 +1,44 @@
+const moment = require('moment');
+
+function dateExpired(regex, file, oldestDate) {
+  const match = regex.exec(file.name);
+  if (match && match[1]) {
+    const date = moment(match[1], 'YYYYMMDD');
+    return date.isBefore(oldestDate);
+  }
+  return false;
+}
+
+function createExpiredDataFilter(outputFile, version, oldestDate) {
+  const regex = new RegExp(`^${outputFile}-(\\d{8})-${version}.json`);
+  return file => dateExpired(regex, file, oldestDate);
+}
+function createExpiredSummaryFilter(outputFile, summaryFile, version, oldestDate) {
```

Comment:
A bit finickity but could the inconsistent spacing be fixed please? Ideally to have a line between each function.